app/models/github_fetcher/resource.rb

```diff
@@ -12,18 +12,24 @@ def initialize(options)
 
     # Generally not over-ridden
     def as_json
-      @as_json ||= response.json_body
+      @as_json ||= Hash.new do |h, key|
+                     h[key] = response.json_body
+                   end
+      @as_json[options]
```

Comment:
I think this is going to blow out memory. For kubernetes we will have 176 keys and just as many bodies in the hash (which are not small). Let's not memoize here.